docs/server/channel.md

```diff
@@ -4,35 +4,46 @@ title: Channel
 sidebar_label: Channel
 ---
 
-`Channel`(aka `Chatroom`) is an important concept when building chat apps. `Channel` can aggregate users and make them prepare chattings. There can exist many relational models in-between `User` and `Channel`. This way makes life easier for a developer who wants to add complex business logic in their chat application.
+This page explains how the `Channels` in `Hackatalk` are structured and was created with the purpose of helping our contributors understand Hackatalk's data structure.
 
-Say you want to make the user grant permission to other users in the `Channel`, then we can create `Membership` relation in between. Another scenario is when the user wants to manage the sound(vibrate, silent, etc) of each channel, we can create `Sound` relation. Like so, we can add more relations in between to handle more complex logic.
+The `Channel`(aka `Chatroom`) relational model is an important concept when it comes to building chatting apps. A `Channel` aggregates users and prepares an environment for chatting.  Many relational models can exist in-between `User` and `Channel`. These in-between relationships make it easier for developers to add complex business logic in their chat applications. Among these many realtionships, we will first look at the `Membership` relationship.
 
-Currently, we've managed to have the above scenario in one relation which is `Membership`. 
+When a user grants permission to other users to access a `Channel`, a `Membership`relationship is created between the `Users`, `Membership`, and `Channel`. The `memberships` table has a `userId` and a `channelId` that links `Users` to their `Channel`.
```

Comment:
based on this diagram below, to my understanding, the 'memberships' relationship links the 'users' and 'channels' through the 'userId' and 'channelId' section. That is why I have described it as above. Just want to check if this is what you were referring to.
<img width="1075" alt="Screen Shot 2020-08-18 at 4 15 08 PM" src="https://user-images.githubusercontent.com/59305253/90482306-fd905180-e16d-11ea-85f7-4d38522551f7.png">